app/views/layouts/home/_navbar_top.html.erb

```diff
@@ -1,27 +1,24 @@
 <!-- Dropdown Structure -->
 <ul id="dropdown-user" class="dropdown-content">
  <% if user_signed_in? %>
+   
   <li>
-
    <%= link_to(teacher_path) do %>
-    <i class="mdi-action-home left"></i> Dashboard
+     <i class="fa fa-home" aria-hidden="true"></i> Dashboard
```

Comment:
The gem font-awesome-rails has a one helper for this.
```Ruby
<%= fa_icon "home", aria-hidden: true %>
```

Reply:
Thanks